src/pages/ClaimsPage/utils.ts

```diff
@@ -62,3 +64,14 @@ export const claimRows = (claims: QueryClaim[]) =>
         : [...finalClaims, curr],
     [] as QueryClaim[]
   );
+
+export const createClaimsRows = (claims: QueryClaim[]) => {
```

Comment:
@levity nice! i like this approach to getting all the data grouped and organized as needed!